zooniverse/front-end-monorepo
# lib-user: add Grommet, zooTheme, storybook, and testing setup

## Package
- lib-user

## Linked Issue and/or Talk Post
Iteration on #5424 .

## Describe your changes
- add eslint config
- add Grommet and zooTheme
- add storybook
- add testing setup

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What user actions should my reviewer step through to review this PR?
  1. run `yarn panic && yarn bootstrap:es6`
  2. from `./packages/lib-user`
  3. run `yarn dev` - confirm dev server runs at `https://local.zooniverse.org:8080/?users=[your staging user login]/stats`
      - note: I often get an error on initial dev server load, though not after page refresh. I'm investigating, but do not have a solution at the moment
  4. run `yarn eslint` - confirm eslint output
  5. run `yarn test` - confirm mocha tests output (more on this in comments)
  6. run `yarn storybook` - confirm storybook at http://localhost:6008/?path=/story/components-groupstats--default
- Are there plans for follow up PR’s to further fix this bug or develop this feature? yes, see linked GitHub Project FEM Stats Efforts

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] You can `yarn panic && yarn bootstrap` and FEM works as expected
